ranger2/aws/ranger_asg/providers.tf

```diff
@@ -0,0 +1,8 @@
+#Configure AWS Provider
+provider "aws" {
+  access_key = var.access_key
+  secret_key = var.secret_key
+  region     = var.region
+  token 	 = var.aws_session_token 
```

Comment:
out of curiosity - i understand that the session tokens are required for devexternalqubole. if the customer using it does not have the same security settings, will this cause any authorization errors
?

Reply:
Fixed add condition